### DISCUSSION TITLE
I need to deploy in windows as raw binary, no docker. Is it possible?

### DISCUSSION THREAD

**Discussion by anborg** [zitadel/zitadel#6061] [Q&A]
I have a constraint, I have to deploy as binary+ config in a windows server. No docker allowed.  Is this source code windows comptible? I did not try compiling yet.

Any suggestions on how I can compile and setup the whole stack for windows? My golang skills are intermediate.

Thank you

P.S: No need to suggest 'go for linux' 'use docker' etc.

**Comment by fforootd** [comment]
As of right now we release the windows version as part of our CI pipeline https://github.com/zitadel/zitadel/releases/tag/v2.28.1

https://github.com/zitadel/zitadel/releases/download/v2.28.1/zitadel_Windows_x86_64.zip

Just be aware that we do not really test Zitadel on windows 😁 

I think with the upcoming changes for our CI https://github.com/zitadel/zitadel/pull/5953 we did not yet include windows.

@adlerhurst right?

**Reply by adlerhurst** [reply]
yep. just added it: https://github.com/zitadel/zitadel/commit/68c0e321abc557a5e35476d9b28be9e8089c3e10
